### ISSUE TITLE
[Accessibility][Panel] Aria controls don't match the element ID

### ISSUE THREAD

**Issue by ndas3** [adobe/aem-core-wcm-components#1208]
Hi Team,

We are facing accessibility issue on core tab and carousel component.

AEM : 6.5 SP5

Steps to replicate:

1. Add core tab and carousel component to a page.
2. Author the components
3. Use accessibility tool (AXE Monitor in our case) to verify the AA compliance of these components.

Error:

Accessibility | Carousel | Invalid ARIA attribute value

Screenshot:
<img width="754" alt="accessibility issue" src="https://user-images.githubusercontent.com/28289196/95754640-f6467980-0cc0-11eb-8682-e330b6cf31f5.PNG">

**Comment by jakublamprecht** [comment]
This happens for both Carousel and Tabs component. Ids that are passed to aria-controls in carousel are not valid ids that are present in the document. In the markup of carousel instead of:
```
aria-controls="${carousel.id}-item-${itemList.index}"
```
there should be:
```
aria-controls="${item.id}"
```
so that the indicator points to the id of tabpanel.

The same goes for tabs, but there's a problem with both aria-controls and aria-labelled by.
Element with role tab has `aria-controls` set to `${tab.id}-tabpanel}` but this id should be given on the `tabpanel` element.  The tabpanel element currently has id set to `${item.id}` instead.
The same goes for `aria-labelledby`. Element with role tabpanel should have `aria-labelledby` value set to the id given on the element with `tab` role.

This mistake in id values also causes w3c errors:
Carousel:
![carouselAriaControls](https://user-images.githubusercontent.com/18114264/97434458-570bbe00-191f-11eb-9632-0644bfb3f6e9.png)
Tabs:
![tabsAriaControls](https://user-images.githubusercontent.com/18114264/97434465-58d58180-191f-11eb-878c-87049cf84c07.png)

**Comment by gabrielwalt** [comment]
This probably affects all panel components: Carousel, Tabs, and Accordions.
